lib/backup/actions/util.go

```diff
@@ -23,6 +23,8 @@ var (
 		"or if both not set, DefaultSharedConfigProfile is used")
 	customS3Endpoint = flag.String("customS3Endpoint", "", "Custom S3 endpoint for use with S3-compatible storages (e.g. MinIO). S3 is used if not set")
 	s3ForcePathStyle = flag.Bool("s3ForcePathStyle", true, "Prefixing endpoint with bucket name when set false, true by default.")
+	s3StorageClass   = flag.String("s3StorageClass", "STANDARD", "The Storage Class that is assigned when objects are uploaded to AWS S3. Supported values are: GLACIER, "+
+		"DEEP_ARCHIVE, GLACIER_IR, INTELLIGENT_TIERING, ONEZONE_IA, OUTPOSTS, REDUCED_REDUNDANCY, STANDARD, STANDARD_IA. Default: STANDARD")
```

Comment:
```suggestion
		"DEEP_ARCHIVE, GLACIER_IR, INTELLIGENT_TIERING, ONEZONE_IA, OUTPOSTS, REDUCED_REDUNDANCY, STANDARD, STANDARD_IA.")
```
Flags library will add `(default: {value})` for flags automatically, so it is not needed to specify it in flag description.

Reply:
Also, it is required to avoid specifying any value by default.
This is required to make it backwards compatible for users who are using S3-compatible systems like MinIO to store their backups. Using `STANDARD` by default might break backup uploads in case their systems won't support same names for Storage Classes.